python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/management-api-models.j2

```diff
@@ -8,16 +8,14 @@
 !
 management api models
 {%     for provider in management_api_models.providers | arista.avd.natural_sort('name') %}
-{%         if provider.name is arista.avd.defined and provider.name | lower in ['smash', 'sysdb'] %}
    !
    provider {{ provider.name }}
-{%             for path in provider.paths | arista.avd.natural_sort('path') %}
-{%                 set provider_cli = "path " ~ path.path %}
-{%                 if path.disabled is arista.avd.defined(true) %}
-{%                     set provider_cli = provider_cli ~ " disabled" %}
-{%                 endif %}
+{%         for path in provider.paths | arista.avd.natural_sort('path') %}
```

Comment:
`path` is also not marked as required or parent.

Reply:
added arista.avd.defined check for management_api_http.providers[].paths[].path`.

Reply:
Reverted this.